shared/desktop/app/node.desktop.tsx

```diff
@@ -211,9 +244,68 @@ const plumbEvents = () => {
           startWinService()
         }
         break
-      case 'requestShowDockIcon':
+      case 'closeWindows': {
```

Comment:
vs before, the main renderer would make the other renderers, but really we want the node thread to do this work.